api-reference-v2/mint.json

```diff
@@ -51,16 +51,16 @@
       ]
     },
     {
-      "group": "Business Profile",
+      "group": "Profile",
       "pages": [
-        "api-reference/business-profile/business-profile--create",
-        "api-reference/business-profile/business-profile--update",
-        "api-reference/business-profile/business-profile--retrieve",
-        "api-reference/business-profile/business-profile--activate-routing-algorithm",
-        "api-reference/business-profile/business-profile--retrieve-active-routing-algorithm",
-        "api-reference/business-profile/business-profile--deactivate-routing-algorithm",
-        "api-reference/business-profile/business-profile--update-default-fallback-routing-algorithm",
-        "api-reference/business-profile/business-profile--retrieve-default-fallback-routing-algorithm"
+        "api-reference/profile/profile--create",
+        "api-reference/profile/profile--update",
+        "api-reference/profile/profile--activate-routing-algorithm",
+        "api-reference/profile/profile--update-default-fallback-routing-algorithm",
+        "api-reference/profile/profile--deactivate-routing-algorithm",
+        "api-reference/profile/profile--retrieve-active-routing-algorithm",
+        "api-reference/profile/profile--retrieve",
+        "api-reference/profile/profile--retrieve-default-fallback-routing-algorithm"
```

Comment:
Is this the order we wish to see in the API reference page? Or should we have create, update, retrieve endpoints first, followed by routing related endpoints?

Reply:
Currently these things are grouped with respect to http methods